ims/src/main/java/com/adobe/aio/util/WorkspaceUtil.java

```diff
@@ -43,7 +43,7 @@ public static Workspace.Builder getSystemWorkspaceBuilder() {
         System.getProperty(Workspace.META_SCOPES),
         System.getProperty(Workspace.PROJECT_ID),
         System.getProperty(Workspace.TECHNICAL_ACCOUNT_ID))) {
-      logger.debug("loading test Workspace from JVM System Properties");
+      logger.info("loading test Workspace from JVM System Properties");
```

Comment:
Do you want to revert the log levels back to DEBUG?

Reply:
yes